fixtures/test_script.py

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/python3
+#tags = ["core_approved", "example",]
+#developer = [ "example", "https://example.org" ]
+#port = "80"
+#call_format = "python3 {{script}} {{ip}} {{port}}"
```

Comment:
Now that I'm thinking about it we should probably make those independent of the script itself since we are preventing other scripts that do not support `#` as a comment from running.

Reply:
For the first release, I think we are good with accepting the files which do accept that as a comment. We can of course expand the parsing of ScriptFiles to include other marks.